<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWindow</class>
 <widget class="QMainWindow" name="DataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>410</width>
      <height>270</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_normal">
     <widget class="QTextEdit" name="text_sendData">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>101</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="bt_sendData">
         <property name="text">
          <string>发送数据</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bt_recevData">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>清除数据</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="text_recevData">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_remind">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>label_remind</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_modbus"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>串口</string>
    </property>
    <addaction name="actionset"/>
    <addaction name="actionReconnect"/>
    <addaction name="actionserialClose"/>
   </widget>
   <widget class="QMenu" name="menuModBus">
    <property name="title">
     <string>ModBus</string>
    </property>
    <addaction name="actionmodeBusAnaly"/>
    <addaction name="actionopenModbus"/>
    <addaction name="actioncloseModbus"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuModBus"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionquit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionset">
   <property name="text">
    <string>设置串口</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="text">
    <string>重新连接</string>
   </property>
  </action>
  <action name="actionserialClose">
   <property name="text">
    <string>关闭串口</string>
   </property>
  </action>
  <action name="actionmodeBusAnaly">
   <property name="text">
    <string>分析Modbus数据</string>
   </property>
  </action>
  <action name="actionopenModbus">
   <property name="text">
    <string>Modbus专注模式</string>
   </property>
  </action>
  <action name="actioncloseModbus">
   <property name="text">
    <string>关闭专注模式</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
